### DISCUSSION TITLE
Multi step simulation

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#2244] [Q&A] [answered]
Hi everyone,
I need to perform a multi-step simulation due to a pre-load condition.
How is possible to create this kind of simulations with Openradioss?

**Comment by MarianBulla** [comment]
Hi FGarelli,
you can make the usage of restart files from a previous simulation and create a 2nd engine file using the previous restart files.
Please, review the manual for "restart" or have a look at the engine file with the "/RUN" keyword for more details.
Hope it helps you ?
Thanks and best regards,
Marian

**Reply by FGarelli** [reply]
Hello Marian, thank you for your guideline.
I manage to restart the simulation with a 2nd engine file but seems that the timestep is not updating. Do you have any suggestion? I attached the running files.
Thanks and kind regards,
Federico
[multi_step.zip](https://github.com/OpenRadioss/OpenRadioss/files/13976025/multi_step.zip)

**Reply by MarianBulla** [reply]
Hello Federico,
I had a deeper look into your model. 
The requests for result files and the timestep limitation is your initial k-file are quite high.
I did a small modification within your model (see attached), and it runs as expected.

Hope it helps you ?

Thanks and best regards,
[mod.zip](https://github.com/OpenRadioss/OpenRadioss/files/14007495/mod.zip)

Marian

**Reply by FGarelli** [reply]
Hi Marian, 
I reviewed the file that you sent me. The main issues of this simulation is that I need to run the first step for *TERMINATION time of 90000ms with a quasi-static simulation (15mm deformation) and a second step with a *TERMINATION time of 7.5ms in an high speed test (150mm deformation).
With the setting that you kindly provide I still use the loading curve for the quasi static step and the simulation ends after 200ms (both steps).

I don't have problems in running but in updating the timestep, which remain stable on the first setting (dt=0.01ms), instead decreasing in the high speed step.
Do you have any suggestions of how to set the simulation, considering these new information?

Federico

**Reply by MarianBulla** [reply]
Hi Federico,
the reason for this high timestep in the 2nd run, is in the amount of added mass in your 1st run. 
It (the high mass on the nodes) remains for the 2nd run.

For such kind of simulations, I usually create state-files from the 1st run, which contain the deformed nodal-coordinates, stresses and strains in the integration poinst, and use them for the 2nd run, which starts from time zero, but deformed structure with initialized stress and strain.
I will test, if this works with the ls-dyna input format, or will try to figure out another solution.

Thanks and best regards,
Marian

**Reply by FGarelli** [reply]
Hello Marian, for further application it will be interested to understand if it's possible to set the simulation from LS-Prepost with a state-file from Openradioss (or any other software) and start a second launch.
I think that in my case, the adding of /MASS/RESET keyword in the engine file results as most efficient.
However, if there's any possibility to verify or contribute to the feasibility of your workflow, I'm available and highly interested.

**Comment by qiangRadioss** [comment, marked as answer]
Hello,

You can add /MASS/RESET (engine input) as well in your *2.rad to not use the added mass from the 1er run.

thanks.
Qiang.

**Reply by FGarelli** [reply]
Thanks Qiang, it works smoothly.

Federico
